packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -94,43 +105,45 @@ String addServiceExtensions() {
         return entry;
       }
     }
-    return null;
+    throw 'Enum value $name not found';
   }
 
   Future<Map<String, dynamic>> getLayoutExplorerNode(
       Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final int subtreeDepth = int.parse(parameters['subtreeDepth']);
-    final String groupName = parameters['groupName'];
-    Map<String, Object> result = {};
+    final id = parameters['id'];
+    final subtreeDepth = int.parse(parameters['subtreeDepth'] as String);
+    final groupName = parameters['groupName'];
+    var result = <String, dynamic>{};
     final instance = WidgetInspectorService.instance;
     final root = instance.toObject(id);
     if (root == null) {
-      result = null;
-    } else {
-      result = instance._nodeToJson(
-        root,
-        InspectorSerializationDelegate(
-            groupName: groupName,
-            summaryTree: true,
-            subtreeDepth: subtreeDepth,
-            service: instance,
-            addAdditionalPropertiesCallback: (node, delegate) {
-              final Map<String, Object> additionalJson = <String, Object>{};
-              final Object value = node.value;
-              if (value is Element) {
-                final renderObject = value.renderObject;
+      return Future.value(<String, dynamic>{
+        'result': result,
+      });
+    }
+    result = instance._nodeToJson(
+      root as DiagnosticsNode,
+      InspectorSerializationDelegate(
+          groupName: groupName,
+          summaryTree: true,
+          subtreeDepth: subtreeDepth,
+          service: instance,
+          addAdditionalPropertiesCallback: (node, delegate) {
+            final Map<String, Object> additionalJson = <String, Object>{};
+            final Object value = node.value as Object;
+            if (value is Element) {
+              final renderObject = value.renderObject;
+              if (renderObject != null) {
                 additionalJson['renderObject'] =
-                    renderObject.toDiagnosticsNode()?.toJsonMap(
+                    renderObject.toDiagnosticsNode().toJsonMap(
                           delegate.copyWith(
                             subtreeDepth: 0,
                             includeProperties: true,
                           ),
-                        );
-                final Constraints constraints = renderObject.constraints;
-                if (constraints != null) {
-                  final Map<String, Object> constraintsProperty =
-                      <String, Object>{
+                        ) as Object;
+                try {
+                  final constraints = renderObject.constraints;
```

Comment:
the constraints property is now non-nullable but could throw an exception if it is accessed at the wrong time..